tests/Processor/SanitizeDataProcessorTest.php

```diff
@@ -125,7 +125,19 @@ public function processDataProvider()
             [
                 [
                     'extra_context' => [
-                        'ccnumba' => '4242424242424242',
+                        'ccnumba' => str_repeat('9', 13),
```

Comment:
Just a minor comment, feel free to ignore it: does it really makes sense to not write the string and instead use `str_repeat`?

Reply:
I did it just for readability, so the length of the test string is immediately clear.